nominees/operation-serenata-de-amor.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "Operation Serenata de Amor",
+  "aliases": [
+    ""
```

Comment:
Can we add Seranata as an alias (the project is referenced as Serenata in multiple places).

Reply:
Should [Rosie](https://twitter.com/RosieDaSerenata), [Jarbas](https://jarbas.serenata.ai/), and [Querido Diário](https://queridodiario.ok.org.br/) be added as aliases as well? If yes, are these specific projects also licensed with open licenses in a public repository?

_Edit_: Looking closer at their [GitHub README](https://github.com/okfn-brasil/serenata-de-amor#how-1), it seems to be answered:

> We have two main repositories [on GitHub](https://github.com/okfn-brasil). This is the main repo and hosts [Rosie](https://github.com/okfn-brasil/serenata-de-amor/blob/main/rosie/README.md) and [Jarbas](https://github.com/okfn-brasil/serenata-de-amor/blob/main/jarbas/README.md). In addition, we have the [toolbox](https://github.com/okfn-brasil/serenata-toolbox) - a pip installable package. Yet there are experimental [notebooks](https://github.com/okfn-brasil/notebooks) maintained by the community and our [static webpage](https://github.com/okfn-brasil/serenata-website).